src/Listener/SentryRequestListener.php

```diff
@@ -0,0 +1,57 @@
+<?php declare(strict_types=1);
+
+namespace Becklyn\Hosting\Listener;
+
+use Sentry\SentryBundle\EventListener\RequestListener;
+use Sentry\SentryBundle\EventListener\RequestListenerControllerEvent;
+use Sentry\SentryBundle\EventListener\RequestListenerRequestEvent;
+use Sentry\SentrySdk;
+use Sentry\State\Scope;
+
+final class SentryRequestListener
+{
+    /**
+     * @var RequestListener
+     */
+    private $inner;
+
+
+    public function __construct (RequestListener $inner)
+    {
+        $this->inner = $inner;
+    }
+
+
+    /**
+     * Set the username from the security context by listening on core.request
+     */
+    public function onKernelRequest (RequestListenerRequestEvent $event) : void
+    {
+        if (!$event->isMasterRequest())
+        {
+            return;
+        }
+
+        $currentClient = SentrySdk::getCurrentHub()->getClient();
+
+        if (null === $currentClient || !$currentClient->getOptions()->shouldSendDefaultPii())
+        {
+            return;
+        }
+
+        $userData = [];
+        $userData['ip_address'] = $event->getRequest()->getClientIp();
```

Comment:
That is not GDPR conform. That is a feature that we explicitly disable in the config, so we don't need to reimplement disabled features.

See https://github.com/Becklyn/Hosting/blob/master/src/DependencyInjection/BecklynHostingExtension.php#L81

Reply:
I removed the content of that function completely.